vocabularies/Common.xml

```diff
@@ -795,8 +795,15 @@ The minimum structure is
         </Property>
       </ComplexType>
       <ComplexType Name="ValueListParameterIn" BaseType="Common.ValueListParameter">
+        <Property Name="Operator" Type="UI.SelectionRangeOptionType" DefaultValue="0" Nullable="false">
+          <Annotation Term="Core.Description" String="Operator applied to the property. Defaults to `EQ`." />
+        </Property>
         <Property Name="LocalDataProperty" Type="Edm.PropertyPath" Nullable="false">
-          <Annotation Term="Core.Description" String="Path to property that is used to filter the value list with `eq` comparison" />
+          <Annotation Term="Core.Description" String="Path to property that is used to filter the value list by applying the operator (Single value or lower interval boundary)" />
+          <Annotation Term="Core.LongDescription" String="In case the property path contains a collection-based navigation or structural property, the filter is a set of `eq` comparisons connected by `or` operators" />
+        </Property>
+        <Property Name="LocalDataPropertyHigh" Type="Edm.PropertyPath" Nullable="false">
```

Comment:
Shouldn't this be nullable? For `Operator = CP` there is no upper boundary.

Reply:
what is meant by operator CP, I have only them https://sapui5.hana.ondemand.com/#/api/sap.ui.model.FilterOperator in mind

Reply:
CP = Contains Pattern

https://github.com/SAP/odata-vocabularies/blob/9c2dc33ba9913d8f1a2d41b3f8223ce204c598e7/vocabularies/UI.xml#L1337-1339

Reply:
Removed the *Between* use case and the upper boundary with 2e24e35.